### ISSUE TITLE
can't install GPy by any way

### ISSUE THREAD

**Issue by YingxiaoKong** [SheffieldML/GPy#684]
Hello,

I have installed GPy before using pip install Gpy. Then By mistake, I delete my anaconda's path, I guess it's still on my computer but I couldn't reach it. So I install a new one. Then the problem came out. I can not install Gpy again by any method. I have tried: 1. pip install GPy(this pip is related to the new anaconda) 2: git clone https://github.com/SheffieldML/GPy.git
cd GPy
git checkout devel
python setup.py build_ext --inplace
nosetests GPy/testing

Neither of them help. The error looks like:
Cloning into 'GPy'...
remote: Enumerating objects: 14, done.
remote: Counting objects: 100% (14/14), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 37648 (delta 5), reused 12 (delta 5), pack-reused 37634
Receiving objects: 100% (37648/37648), 61.79 MiB | 5.25 MiB/s, done.
Resolving deltas: 100% (26297/26297), done.
yingxiaomatoMacBook-Pro:Documents yingxiaokong$ cd GPy
yingxiaomatoMacBook-Pro:GPy yingxiaokong$ ls
AUTHORS.txt	LICENSE.txt	appveyor.yml	doc		travis_tests.py
CHANGELOG.md	MANIFEST.in	benchmarks	setup.cfg
GPy		README.md	codecov.yml	setup.py
yingxiaomatoMacBook-Pro:GPy yingxiaokong$ git checkout devel
Already on 'devel'
Your branch is up-to-date with 'origin/devel'.
yingxiaomatoMacBook-Pro:GPy yingxiaokong$ ptyhon setup.py build_ext --inplace
-bash: ptyhon: command not found
yingxiaomatoMacBook-Pro:GPy yingxiaokong$ python setup.py build_ext --inplace
running build_ext
building 'GPy.kern.src.stationary_cython' extension
creating build
creating build/temp.macosx-10.7-x86_64-3.7
creating build/temp.macosx-10.7-x86_64-3.7/GPy
creating build/temp.macosx-10.7-x86_64-3.7/GPy/kern
creating build/temp.macosx-10.7-x86_64-3.7/GPy/kern/src
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/yingxiaokong/anaconda3/include -arch x86_64 -I/Users/yingxiaokong/anaconda3/include -arch x86_64 -I/Users/yingxiaokong/anaconda3/lib/python3.7/site-packages/numpy/core/include -I. -I/Users/yingxiaokong/anaconda3/include/python3.7m -c GPy/kern/src/stationary_cython.c -o build/temp.macosx-10.7-x86_64-3.7/GPy/kern/src/stationary_cython.o -O3
In file included from GPy/kern/src/stationary_cython.c:242:
In file included from /Users/yingxiaokong/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4:
In file included from /Users/yingxiaokong/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:18:
In file included from /Users/yingxiaokong/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1823:
/Users/yingxiaokong/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: 
      "Using deprecated NumPy API, disable it by "          "#defining
      NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
#warning "Using deprecated NumPy API, disable it by " \
 ^
GPy/kern/src/stationary_cython.c:2271:21: warning: code will never be executed
      [-Wunreachable-code]
        if (1 == 0) abort();
                    ^~~~~
GPy/kern/src/stationary_cython.c:2271:13: note: silence by adding parentheses to
      mark code as explicitly dead
        if (1 == 0) abort();
            ^
            /* DISABLES CODE */ ( )
GPy/kern/src/stationary_cython.c:4234:134: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_2++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[1]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:5456:134: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_1++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:7433:138: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_3++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:7425:137: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_3++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:10966:138: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_5++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:10958:137: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_5++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:11583:138: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_7++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:11575:137: warning: code will never be executed
      [-Wunreachable-code]
  ...__pyx_t_7++; if (unlikely(0 < 0)) {__pyx_filename = __pyx_f[2]; __pyx_li...
                                                         ^~~~~~~
GPy/kern/src/stationary_cython.c:19389:21: error: no member named 'exc_type' in
      'struct _ts'
    *type = tstate->exc_type;
            ~~~~~~  ^
GPy/kern/src/stationary_cython.c:19390:22: error: no member named 'exc_value' in
      'struct _ts'; did you mean 'curexc_value'?
    *value = tstate->exc_value;
                     ^~~~~~~~~
                     curexc_value
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:237:15: note: 
      'curexc_value' declared here
    PyObject *curexc_value;
              ^
GPy/kern/src/stationary_cython.c:19391:19: error: no member named
      'exc_traceback' in 'struct _ts'; did you mean 'curexc_traceback'?
    *tb = tstate->exc_traceback;
                  ^~~~~~~~~~~~~
                  curexc_traceback
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:238:15: note: 
      'curexc_traceback' declared here
    PyObject *curexc_traceback;
              ^
GPy/kern/src/stationary_cython.c:19403:24: error: no member named 'exc_type' in
      'struct _ts'
    tmp_type = tstate->exc_type;
               ~~~~~~  ^
GPy/kern/src/stationary_cython.c:19404:25: error: no member named 'exc_value' in
      'struct _ts'; did you mean 'curexc_value'?
    tmp_value = tstate->exc_value;
                        ^~~~~~~~~
                        curexc_value
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:237:15: note: 
      'curexc_value' declared here
    PyObject *curexc_value;
              ^
GPy/kern/src/stationary_cython.c:19405:22: error: no member named
      'exc_traceback' in 'struct _ts'; did you mean 'curexc_traceback'?
    tmp_tb = tstate->exc_traceback;
                     ^~~~~~~~~~~~~
                     curexc_traceback
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:238:15: note: 
      'curexc_traceback' declared here
    PyObject *curexc_traceback;
              ^
GPy/kern/src/stationary_cython.c:19406:13: error: no member named 'exc_type' in
      'struct _ts'
    tstate->exc_type = type;
    ~~~~~~  ^
GPy/kern/src/stationary_cython.c:19407:13: error: no member named 'exc_value' in
      'struct _ts'; did you mean 'curexc_value'?
    tstate->exc_value = value;
            ^~~~~~~~~
            curexc_value
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:237:15: note: 
      'curexc_value' declared here
    PyObject *curexc_value;
              ^
GPy/kern/src/stationary_cython.c:19408:13: error: no member named
      'exc_traceback' in 'struct _ts'; did you mean 'curexc_traceback'?
    tstate->exc_traceback = tb;
            ^~~~~~~~~~~~~
            curexc_traceback
/Users/yingxiaokong/anaconda3/include/python3.7m/pystate.h:238:15: note: 
      'curexc_traceback' declared here
    PyObject *curexc_traceback;
              ^
GPy/kern/src/stationary_cython.c:19451:24: error: no member named 'exc_type' in
      'struct _ts'
    tmp_type = tstate->exc_type;


Did anyone has same problem as me?

**Comment by vlavorini** [comment]
Yes, I have the same problem. In https://github.com/SheffieldML/GPy/issues/649 they claim to have solved, but following those instructions I still have some issue. But you can give a try.
